country-info.csv

```diff
@@ -391,6 +391,7 @@ Uppsala University,europe,se
 Utrecht University,europe,nl
 VU Amsterdam,europe,nl
 Victoria University of Wellington,australasia,nz
+Vidyasirimedhi Institute of Science and Technology,asia,th

```

Comment:
This name is too long. Please rename to VISTEC.

Reply:
Thank you very much for reviewing this commit. The institution's name has been renamed to VISTEC according to your suggestion.